docs/user_guide/configuration/index.md

```diff
@@ -0,0 +1,304 @@
+
+### What is a Broker?
+
+A broker is a message queue that acts as an intermediary between the sender of a task and the worker processes that execute the task. It facilitates the communication between different parts of a distributed system by passing messages (tasks) from producers (the code that generates tasks) to consumers (worker processes that execute tasks).
+
+The broker is responsible for queuing the tasks and delivering them to the appropriate worker processes. It allows for the decoupling of the task producer and the task consumer, enabling a more scalable and flexible architecture.
+
+[Celery supports various message brokers](https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/index.html), including [RabbitMQ](https://www.rabbitmq.com/), [Redis](https://redis.io/), and others. You can configure Celery to use a specific broker based on your requirements (although we suggest using RabbitMQ).
+
+See the [Configuring the Broker and Results Backend](#configuring-the-broker-and-results-backend) section below for more information on configuring your broker.
+
+### What is a Results Backend?
+
+The results backend is a storage system where the results of executed tasks are stored. After a task is executed by a worker, the result is stored in the result backend, and the original task sender can retrieve the result later.
+
+The results backend enables the asynchronous nature of Celery. Instead of blocking and waiting for a task to complete, the sender can continue with other work and later retrieve the result from the results backend.
+
+[Celery supports various results backends](https://docs.celeryq.dev/en/stable/getting-started/backends-and-brokers/index.html), including databases (such as SQLAlchemy, Django ORM), message brokers (Redis, RabbitMQ), and others. You can configure Celery to use a specific broker based on your requirements (although we suggest using Redis). However, since Merlin utilizes [Celery chords](https://docs.celeryq.dev/en/stable/userguide/canvas.html#chords) and the amqp (rpc RabbitMQ) server does not support chords, we cannot use RabbitMQ as a results backend.
+
+See the [Configuring the Broker and Results Backend](#configuring-the-broker-and-results-backend) section below for more information on configuring your results backend.
+
+## The app.yaml File
+
+In order to read in configuration options for your Celery settings, broker, and results backend, Merlin utilizes an app.yaml file.
+
+There's a built-in command with Merlin to set up a skeleton app.yaml for you:
+
+```bash
+merlin config
+```
+
+This command will create an app.yaml file in the `~/.merlin/` directory that looks like so:
+
+```yaml
```

Comment:
Is there a way to make this dynamic, eg by pointing to the actual file that will get created, so if we change the source it stays up to date?

Reply:
I did some digging this morning and I think I found a plugin for something like this. I'll try it out and swap out code blocks like this.